RecoVertex/BeamSpotProducer/src/PVFitter.cc

```diff
@@ -87,6 +86,8 @@ void PVFitter::initialize(const edm::ParameterSet& iConfig,
   errorScale_        = iConfig.getParameter<edm::ParameterSet>("PVFitter").getUntrackedParameter<double>("errorScale");
   sigmaCut_          = iConfig.getParameter<edm::ParameterSet>("PVFitter").getUntrackedParameter<double>("nSigmaCut");
   fFitPerBunchCrossing=iConfig.getParameter<edm::ParameterSet>("PVFitter").getUntrackedParameter<bool>("FitPerBunchCrossing");
+  useOnlyFirstPV_    = iConfig.getParameter<edm::ParameterSet>("PVFitter").getUntrackedParameter<bool>("useOnlyFirstPV");
+  minSumPt_          = iConfig.getParameter<edm::ParameterSet>("PVFitter").getUntrackedParameter<double>("minSumPt");
```

Comment:
@perrotta thanks a lot for the reply and the explanation.
But again, if you take a look at the current version of the PVFitter in the official CMSSW master branch (https://github.com/cms-sw/cmssw/blob/master/RecoVertex/BeamSpotProducer/src/PVFitter.cc), all these parameters are untracked. Even if you look back in the history of the file, you can see the parameters were untracked since 2010 (https://github.com/cms-sw/cmssw/blob/ab40b3502fe7dfa3a4039a33cc29539456b594d3/RecoVertex/BeamSpotProducer/src/PVFitter.cc)